### PR TITLE
Updated user-namespaces.md

### PR DESCRIPTION
**Pull request by YashPimple**
Resolve the small issue in the lines 92 and 94

Issue #36738

### CODE REVIEW

File: content/en/docs/concepts/workloads/pods/user-namespaces.md [hunk 1]
```diff
@@ -89,10 +89,9 @@ isolate the users in the container from the users in the node.
 This means containers can run as root and be mapped to a non-root user on the
 host. Inside the container the process will think it is running as root (and
 therefore tools like `apt`, `yum`, etc. work fine), while in reality the process
-doesn't have privileges on the host. You can verify this, for example, if you
-check the user the container process is running `ps` from the host. The user
-`ps` shows is not the same as the user you see if you execute inside the
-container the command `id`.
+doesn't have privileges on the host. You can verify this, for example, if you check
+the user the container process is running `ps` from the host. The user in the output
+of the `ps` command is not the same as you execute the command `id` inside the container.
```
Comment (tengqm) [inline content/en/docs/concepts/workloads/pods/user-namespaces.md:94]: The revised version is still unclear.

**Comment by natalisucks** [comment]
@YashPimple Hi there, are you able to address the comments from other reviewers so we can move this PR forward? Thanks!

**Comment by Kartik494** [comment]
@YashPimple could you addressed the comments and rebase  the PR.

**Comment by natalisucks** [comment]
Given there's been little movement on addressing the feedback on this PR, I'll be closing it. However, @YashPimple, if you still want to work on this, please feel free to re-open and address the comments. Thanks muchly 🤝 
/close

**Pull request closed without merging**